src/sentry/static/sentry/app/components/events/contexts/state.tsx

```diff
@@ -20,23 +21,40 @@ type Props = {
 };
 
 class StateContextType extends React.Component<Props> {
+  getStateTitle(name: string, type?: string) {
+    return name + (type ? ` (${upperFirst(type)})` : '');
+  }
+
   getKnownData(): KeyValueListData[] {
     const primaryState = this.props.data.state;
 
     return [
       {
         key: 'state',
-        subject:
-          t('State') + (primaryState.type ? ` (${upperFirst(primaryState.type)})` : ''),
+        subject: this.getStateTitle(t('State'), primaryState.type),
         value: primaryState.value,
       },
     ];
   }
 
+  getUnknownData(): KeyValueListData[] {
+    const {data} = this.props;
+
+    return Object.entries(data)
+      .filter(([key]) => !['type', 'title', 'state'].includes(key))
```

Comment:
Hmm please check. This filters out `'type', 'title', 'state'`, in `getUnknownData.tsx` with filter out only `type` and `title`.

Maybe they're indeed meant to be different. I'm not sure what is this `state` context.

Reply:
This is correct, `state` is being displayed in a knownData section. (and 'type', 'title' are always omitted)
This new state context was merged earlier today, here is PR for updating the docs: https://github.com/getsentry/develop/pull/107